extensions/namespace-lookup/src/main/java/io/druid/server/namespace/cache/NamespaceExtractionCacheManager.java

```diff
@@ -315,7 +319,7 @@ private boolean removeNamespaceLocalMetadata(final NamespaceImplData implDatum)
   }
 
   // Optimistic scheduling of updates to a namespace.
-  public <T extends ExtractionNamespace> ListenableFuture<?> schedule(final T namespace)
+  public <T extends ExtractionNamespace> Future<?> schedule(final T namespace)
```

Comment:
any reason we are changing this to a regular Future? We should be using ListenableFuture everywhere if possible.